sql_generators/serp_events_v2/templates/view.sql

```diff
@@ -0,0 +1,8 @@
+--- User-facing view. Generated via sql_generators.active_users.
+CREATE OR REPLACE VIEW
+  `{{ project_id }}.{{ app_name }}.serp_events_v2`
```

Comment:
You want to create a separate view as well and then later repoint?

Reply:
You're right, not sure why I added this. I'll remove.

Reply:
You're right, not sure why I added this. I'll remove.